src/main/scala/com/databricks/labs/overwatch/validation/Kitana.scala

```diff
@@ -367,6 +367,7 @@ object Kitana {
       databaseName = Some(snapDBName),
       databaseLocation = snapDBLocation,
       etlDataPathPrefix = snapDBDataLocation,
+//      catalogName =  Some("overwatch"),
```

Comment:
remove this commented line if not needed.

Reply:
removed